templates/.vscode/tasks.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "2.0.0",
+  "problemMatcher": "$tsc-watch",
+  "tasks": [
+    {
+      "label": "Compile",
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "command": "yarn",
+      "type": "shell",
+      "presentation": {
+        "focus": false,
+        "panel": "dedicated"
+      },
+      "args": ["run", "prepack"],
+      "isBackground": false,
+      "problemMatcher": {
+        "owner": "typescript",
+        "fileLocation": "relative",
+        "pattern": {
+          "regexp": "^(.*\\.ts):(\\d*):(\\d*)(\\s*-\\s*)(error|warning|info)\\s*(TS\\d*):\\s*(.*)$",
+          "file": 1,
+          "line": 2,
+          "column": 3,
+          "severity": 5,
+          "code": 6,
+          "message": 7
+        }
+      }
+    },
+    {
+      "label": "Lint",
```

Comment:
Similar to the `Compile` task, this one will run the new `lint` script provided out-of-the-box by the generator and will display any errors in VSCode's `Problem View`.
![Screen Shot 2019-07-16 at 3 20 28 PM](https://user-images.githubusercontent.com/1041842/65047682-b8d5b800-d917-11e9-9fae-e5e13f7626ac.png)